www/examples/example13-transform-transition.html

```diff
@@ -1,7 +1,8 @@
 <link rel="stylesheet" href="example13-transform-transition.css">
 <body style="scroll-behavior:smooth">
+<div style="height:100px"></div>
```

Comment:
Can we make this work without `height` using, like, `<br>` and some small character like a period?